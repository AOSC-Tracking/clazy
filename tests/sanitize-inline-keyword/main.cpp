class MyClass
{
    inline int a();
    inline int b();
    int c();
    int d();
};

inline int MyClass::a()
{
    return 7;
}

int MyClass::b()
{
    return 3;
}

inline int MyClass::c()
{
    return 5;
}

// inline not the first Token on the line
int inline MyClass::d()
{
    return 9;
}

// Checks only exported classes, i.e. visibility "default"
class __attribute__((visibility("hidden"))) MyHiddenClass
{
    inline int a();
    inline int b();
    int c(); // Left as-is
};

inline int MyHiddenClass::a()
{
    return 7;
}

int MyHiddenClass::b()
{
    return 3;
}

inline int MyHiddenClass::c()
{
    return 5;
}
