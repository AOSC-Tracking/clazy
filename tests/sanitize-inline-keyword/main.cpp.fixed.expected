class MyClass
{
    inline int a();
    inline int b();
    inline int c();
    inline int d();
};

inline int MyClass::a()
{
    return 7;
}

int MyClass::b()
{
    return 3;
}

 int MyClass::c()
{
    return 5;
}

// inline not the first Token on the line
int  MyClass::d()
{
    return 9;
}

// Checks only exported classes, i.e. visibility "default"
class __attribute__((visibility("hidden"))) MyHiddenClass
{
    inline int a();
    inline int b();
    int c(); // Left as-is
};

inline int MyHiddenClass::a()
{
    return 7;
}

int MyHiddenClass::b()
{
    return 3;
}

inline int MyHiddenClass::c()
{
    return 5;
}
